charactersheet.html

```diff
@@ -0,0 +1,579 @@
+<!DOCTYPE html>
+<html lang="en">
+<!--head of the html page-->
+
+<head>
+    <!--meta information-->
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <!-- Compiled and minified CSS -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css">
+
+    <!-- Compiled and minified JavaScript -->
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans&family=Pirata+One&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+        integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous" />
+        <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=MedievalSharp&display=swap" rel="stylesheet">  
+    <style>
+        @media only screen and (min-width: 993px) {
+            .container {
+                width: 100%;
+            }
+        }
+    </style>
+</head>
+
+<body>
+    <header>
+
+    <!--Save Button-->
+    <a class="waves-effect waves-light btn" id="save-button">Save Character</a>
+    <!--DICE-->
+    <div id="click-to-roll" class="modal-trigger" href="#diceResult">
+        <p id=clickroll>Click to roll!</p>
+        <i class="fas fa-dice-d20" id="roll20"></i>
+    </div>
+    <!--Dice Roll Modal-->
+    <div class="modal" id="diceResult" style="font-size: 50px;">
+    </div>
+    <!--Adventure Companion Heading-->
+    <div class="container">
+        <div class="row">
```

Comment:
Try to auto format your code, these indentations don't match up and they make it hard to read